jpo-asn-runtime/src/main/java/us/dot/its/jpo/asn/runtime/types/Asn1Sequence.java

```diff
@@ -0,0 +1,32 @@
+package us.dot.its.jpo.asn.runtime.types;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import us.dot.its.jpo.asn.runtime.serialization.SerializationUtil;
+
+@Slf4j
+public abstract class Asn1Sequence implements Asn1Type {

```

Comment:
suggestion: comments to clarify functionality. For instance, what is "extensionMarker"?

Reply:
question: is this class just a wrapper for a boolean? I'm very confused what we're doing with this class

Reply:
> question: is this class just a wrapper for a boolean? I'm very confused what we're doing with this class

Asn1Sequence is the base class for all ASN.1 SEQUENCE types.  It has nothing to do with BOOLEAN types.

Reply:
> suggestion: comments to clarify functionality. For instance, what is "extensionMarker"?

It indicates if the extension marker is present in the specification for the type.  For example this type is a SEQUENCE with an extension maker (the three dots):

```
PathPrediction ::= SEQUENCE {
   radiusOfCurve RadiusOfCurvature, 
   confidence    Confidence, 
   ...  
   }
```

Reply:
> > question: is this class just a wrapper for a boolean? I'm very confused what we're doing with this class
> 
> Asn1Sequence is the base class for all ASN.1 SEQUENCE types. It has nothing to do with BOOLEAN types.

I guess the confusion is because it contains the hasExtensionMarker field which is boolean.  But it is mostly just a marker class, to mark that classes derived from it are SEQUENCE types.

Reply:
Added class-level javadocs to all base types, and to extension properties.